src/Acuminator/Acuminator.Analyzers/StaticAnalysis/CallingBaseDataViewDelegate/CallingBaseDataViewDelegateFromOverrideDelegateAnalyzer.cs

```diff
@@ -18,7 +18,9 @@ public class CallingBaseDataViewDelegateFromOverrideDelegateAnalyzer : IPXGraphA
         public ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics =>
             ImmutableArray.Create(Descriptors.PX1087_CausingStackOverflowExceptionInBaseViewDelegateInvocation);
 
-        public void Analyze(SymbolAnalysisContext context, PXContext pxContext, CodeAnalysisSettings settings, PXGraphSemanticModel pxGraph)
+		public virtual bool ShouldAnalyze(PXContext pxContext, CodeAnalysisSettings settings) => true;
```

Comment:
I think it is better to add a base class with this method returning true instead of copy-pasting it in most of the analyzers

Reply:
Yes, I agree. I'll think about a best way to add base aggregated analyzer class to system